src/context.rs

```diff
@@ -76,6 +76,8 @@ struct ConcreteBaseAudioContextInner {
     queued_messages: Mutex<Vec<ControlMessage>>,
     /// number of frames played
     frames_played: Arc<AtomicU64>,
+    /// control msg to add the AudioListener, to be sent when the first panner is created
+    inactive_audio_listener: Mutex<Vec<ControlMessage>>,
```

Comment:
I find the naming a bit confusing, `audio_listener_messages`, `audio_listener_registration_messages`?